core/src/types/operator/operator.rs

```diff
@@ -1636,6 +1719,24 @@ impl Operator {
     /// # }
     /// ```
     ///
+    /// ## `version`
+    ///
+    /// Specify whether to list all object versions or not
```

Comment:
The behavior we implemented is "list files along with their versions." Please always remember that OpenDAL's vision is "accessing data freely." We should do our best to distinguish OpenDAL's API from the underlying storage services' implementation details.

Reply:
how about
```
    /// ## `version`
    ///
    /// Specify whether to list files along with all their versions or not
    ///
    /// if `version` is not enabled, or is set to false, only the current files will be returned.
    ///
    /// if `version` is set to true, all the file versions will be returned.
```